src/engine/Resources/Sound.ts

```diff
@@ -274,8 +276,15 @@ export class Sound implements ILoadable, IAudio {
 
    /**
     * Play the sound, returns a promise that resolves when the sound is done playing
+    * Valid string inputs include:
+    * -'linearUp'
+    * -'linearDown'
+    * -'ease'
+    * -'ease-in'
+    * -'ease-out'
+    * -'ease-in-out'
     */
-   public play(volume?: string): Promise<boolean> {
+   public play(data?: string | number | IBezierPoints): Promise<boolean> {
```

Comment:
Right now, I've got it so that the bezier points will be set automatically if the type of the argument passed into `play` is `string`. If we make it so that 'linearUp', 'ease', etc. are all their own types, is there a way to group those types together at runtime, so that we know the input wasn't an `IBezierPoints`?

Essentially I'm saying the syntax `typeof variable == 'string'` works, but `typeof variable == "IBezierPoints"` and `variable instanceof IBezierPoints` throw errors, so I'm not sure how to identify custom types/interfaces at runtime.